### ISSUE TITLE
Value is not nullified when deleting the value via highlighting

### ISSUE THREAD

**Issue by ademirsoy** [pzuraq/ember-inputmask#4]
Hi,

I'm using number-input and only when I first open the page and delete the existing value of the field via highlighting it all, then the unmaskedValue is not nullified and remains the same. 

Have you ever come accross this issue?

This is how I use the component
{{number-input unmaskedValue=minTradeQty.value group=true groupSize=3 separator="." decimal=false radix=","}}

Thanks!
Ali

**Comment by pzuraq** [comment]
Haven't encountered this before, I'll take a look.

To be clear, you are selecting the contents of the input field and deleting them by pressing delete?

**Comment by ademirsoy** [comment]
Yep, but it only happens when I'm on an edit page where there is already a value, and I delete it without typing anything.

**Comment by oswaldoacauan** [comment]
Same issue here.

**Comment by blimmer** [comment]
So the issue is if the value is prepopulated, you highlight the value with your mouse and hit the "Backspace" key?

**Comment by oswaldoacauan** [comment]
@blimmer exactly

**Comment by bookergabeurena** [comment]
Same issue here...

Highlighting full content of field and pressing BACKSPACE/DELETE triggers observer when **value** property is used:

	{{!-- file.hbs --}}
	{{phone-number-input value=customerCellPhone}}
	
	/* component.js */
	customerCellPhoneChanged: Ember.observer('customerCellPhone', function() {
		console.log('customerCellPhoneChanged:', this.get('customerCellPhone'));
	})
	
Same steps above but using **unmaskedValue** does not trigger observer:

	{{!-- file.hbs --}}
	{{phone-number-input unmaskedValue=customerCellPhone}}
	
	/* component.js */
	customerCellPhoneChanged: Ember.observer('customerCellPhone', function() {
		console.log('customerCellPhoneChanged:', this.get('customerCellPhone'));
	})

**Comment by mattmcginnis** [comment]
Anyone ever find a work-around for this?

**Comment by mattmcginnis** [comment]
FYI, here's a hack that works:

```javascript
    keyUp: _ember['default'].observer('value', function () {
      this.set('unmaskedValue', this.$().inputmask('unmaskedvalue'));
    }),
```

**Comment by brandynbennett** [comment]
When I try this with `one-way-number-mask`, the value is not null but an empty string. Is that better than before?
